src/contexts/user.js

```diff
@@ -0,0 +1,13 @@
+import { createContext, useState } from "react";
+
+export const UserContext = createContext()
+
+export const UserProvider = ({children}) => {
+    const [username, setUsername] = useState(null)
```

Comment:
I think user is a better state name here considering it is an object that **contains** username